build.gradle

```diff
@@ -40,13 +40,13 @@ download {
 	onlyIfNewer = true
 	overwrite = true
     src 'http://cruise.eecs.uottawa.ca/umpleonline/scripts/umple.jar'
-    dest '../../libs/'
+    dest '/libs/umple-latest.jar'
```

Comment:
Not sure this should be `/libs`, shouldn't it just be `libs/` (haven't tried it yet).